tools/topology/topology2/cavs-mixin-mixout-hda.conf

```diff
@@ -17,6 +17,7 @@ IncludeByKey.DEEPBUFFER_FW_DMA_MS{
 Object.Dai {
 	HDA.0 {
 		name $HDA_ANALOG_DAI_NAME
```

Comment:
"This is required because the topology2 compiler allows for expansion of string values with variable definitions." - did you mean "doesn't allow?" Same in the next commit. But how does it help to solve the issue with included topology2 files? You still need to use unique module / component names in them, don't you? And since you don't know where it's included from and what numbers are already taken you end up trying to guess some "really large number that nobody hopefully has taken yet"

Reply:
@lyakh  no I meant "allow". This PR is not meant to solve the issue with topology2 files but rather only amined at being able to allow substituting the dai_index for DAI objects and name for hw_config objects